testing/endpoint.go

```diff
@@ -473,26 +475,35 @@ func (endpoint *Endpoint) ChanCloseInit() error {
 // The packet sequence generated for the packet to be sent is returned. An error
 // is returned if one occurs.
 func (endpoint *Endpoint) SendPacketV2(
-	timeoutHeight clienttypes.Height,
 	timeoutTimestamp uint64,
-	version string,
-	data []byte,
+	packetData ...channeltypesv2.PacketData,
 ) (uint64, error) {
-	// no need to send message, acting as a module
-	sequence, err := endpoint.Chain.App.GetPacketServer().SendPacket(endpoint.Chain.GetContext(), endpoint.ChannelID, endpoint.ChannelConfig.PortID, endpoint.Counterparty.ChannelConfig.PortID, timeoutHeight, timeoutTimestamp, version, data)
+	msg := channeltypesv2.NewMsgSendPacket(endpoint.ChannelID, timeoutTimestamp, endpoint.Chain.SenderAccount.GetAddress().String(), packetData...)
+
+	res, err := endpoint.Chain.SendMsgs(msg)
 	if err != nil {
 		return 0, err
 	}
 
-	// commit changes since no message was sent
-	endpoint.Chain.Coordinator.CommitBlock(endpoint.Chain)
-
 	err = endpoint.Counterparty.UpdateClient()
 	if err != nil {
 		return 0, err
 	}
 
-	return sequence, nil
+	// get sequence from msg response
+	var msgData sdk.TxMsgData
```

Comment:
bit annoying but don't know a better way and relayers will have to do this effectively